doc/man1/openssl-enc.pod

```diff
@@ -6,37 +6,38 @@ openssl-enc - symmetric cipher routines
 
 =head1 SYNOPSIS
 
-B<openssl enc -I<cipher>>
+B<openssl>
+[B<->I<cipher>]
```

Comment:
Again

Reply:
again, disagree.  someone can fix that in a separate PR.

Reply:
So we're going to have you "fixing", just for someone else to "unfix"?

Reply:
> So we're going to have you "fixing", just for someone else to "unfix"?

It won't be the first time and it most definitely won't be the last.

The changes are consistent and I consider that more important.  Others may disagree, that's okay.